solutions/secure-regional-bucket/variables.tf

```diff
@@ -0,0 +1,172 @@
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The IBM Cloud API Token"
+  sensitive   = true
+}
+
+variable "region" {
+  description = "The region to provision the IBM Cloud Object Storage regional bucket is provisioned."
+  type        = string
+  default     = "us-south"
+}
+
+variable "add_bucket_name_suffix" {
+  type        = bool
+  description = "Add random generated suffix (4 characters long) to the newly provisioned COS bucket name (Optional)."
+  default     = true
+}
+
+variable "existing_cos_instance_id" {
+  description = "The ID of an existing Cloud Object Storage instance. Required if 'var.create_cos_instance' is false."
+  type        = string
+}
+
+variable "bucket_access_tags" {
+  type        = list(string)
+  description = "A list of access tags to apply to the Cloud Object Storage instance bucket."
+  default     = []
+}
+
+variable "bucket_name" {
+  type        = string
+  description = "The name to give the newly provisioned COS bucket. Only required if 'create_cos_bucket' is true."
+  default     = "regional-bucket"
```

Comment:
remove default value and make this a required input since it has to be globally unique name